lib/modules/platform/bitbucket/index.ts

```diff
@@ -365,13 +365,17 @@ async function getStatus(
   memCache = true
 ): Promise<BitbucketStatus[]> {
   const sha = await getBranchCommit(branchName);
-  return utils.accumulateValues<BitbucketStatus>(
-    // TODO: types (#7154)
-    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-    `/2.0/repositories/${config.repository}/commit/${sha}/statuses`,
-    'get',
-    { memCache }
-  );
+  return (
+    await bitbucketHttp.getJson<PagedResult<BitbucketStatus>>(
+      // TODO: types (#7154)
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      `/2.0/repositories/${config.repository}/commit/${sha}/statuses`,
```

Comment:
```suggestion
      `/2.0/repositories/${config.repository}/commit/${!sha}/statuses`,
```

Reply:
the `!` needs no be after `sha` 🙃

Reply:
At least locally tslint does not complain about it 🤷🏻‍♂️

Reply:
if it's before then it'll negate the expression. so it would be a bool which is allowed inside the template

Reply:
I shouldn't review before my first coffee 😅